src/include/utime.h

```diff
@@ -239,14 +239,14 @@ class utime_t {
       out << (long)sec() << "." << std::setw(6) << usec();
     } else {
       // this looks like an absolute time.
-      //  aim for http://en.wikipedia.org/wiki/ISO_8601
+      //  conform to http://en.wikipedia.org/wiki/ISO_8601
```

Comment:
You might want to use the routines in common/iso_8601.h

Reply:
that'll work for gmtime(), but not the others.  the parser is tempting except it doesn't support the timezone portion.